src/omnisharpDownload.ts

```diff
@@ -45,46 +46,39 @@ function getOmnisharpAssetName(): string {
     }
 }
 
+function download(urlString: string): Promise<stream.Readable> {
+    let url = parse(urlString);
+    let options: https.RequestOptions = {
+        host: url.host,
+        path: url.path,
+    }
+    
+    return new Promise<stream.Readable>((resolve, reject) => {
+        return https.get(options, res => {
+            // handle redirection
+            if (res.statusCode === 302) {
+                return download(res.headers.location);
+            }
+            else if (res.statusCode !== 200) {
+                return reject(Error(`Download failed with code ${res.statusCode}.`));
+            }
+            
+            return resolve(res);
+        });
+    });
+}
+
 export function downloadOmnisharp(): Promise<boolean> {
     return new Promise<boolean>((resolve, reject) => {
         console.log(`[OmniSharp]: Installing to ${DefaultInstallLocation}`);
         
-        const repo = new Github({ repo: OmnisharpRepo, token: ApiToken });
         const assetName = getOmnisharpAssetName();
+        const urlString = `${BaseDownloadUrl}/${assetName}`;
```

Comment:
I am not sure if we really want to fix this since it is already published to blob storage, but should there be a version number as part of this URL?